### PR TITLE
Check 71 - Create organization creation form on admin site

### PR DESCRIPTION
**Pull request by digitalsoba**
# Description
- Installed Vuetify
- Added Nav bar
- Added Signup page
- Update router

## Type of change

Please delete options that are not relevant.

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

# How Has This Been Tested?

- Load up your environment `docker-compose up -d`
- Run yarn
- Run `yarn run dev`
- Visit [http://localhost:8080](localhost:8080) and verify the sign up page matches the paper mockup

# Checklist:

- [ ] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes

### CODE REVIEW

**Review by 404tonynotfound — APPROVED** [review]

File: resources/js/router/index.js [hunk 1]
```diff
@@ -2,12 +2,21 @@ import Vue from 'vue';
 import VueRouter from 'vue-router';
 import store from '../store';
 
+import SignUp from '../views/SignUp.vue'
```
Comment (404tonynotfound) [inline resources/js/router/index.js]: Can we put the signup component in router/views/signup/index.vue

File: resources/js/views/SignUp.vue [hunk 2]
```diff
@@ -0,0 +1,58 @@
...
+        </v-flex>
+      </v-layout>
+    </v-container>
+  </v-content>
+</template>
+
+<script>
+export default {
+  data: () => ({
+    organization: "",
```
Comment (404tonynotfound) [inline resources/js/views/SignUp.vue]: we should make these null by default instead of empty strings

File: resources/js/views/SignUp.vue [hunk 3]
```diff
@@ -0,0 +1,58 @@
...
+    email: "",
+    password: "",
+    validatepassword: "",
+    address1: "",
+    address2: "",
+    city: "",
+    country: "",
+    zip: ""
+  }),
+  name: "SignUp"
```
Comment (404tonynotfound) [inline resources/js/views/SignUp.vue]: name is usually not necessary if the filename is the same

File: resources/sass/_variables.scss [hunk 4]
```diff
@@ -3,9 +3,9 @@
 $body-bg: #f8fafc;
 
 // Typography
-$font-family-sans-serif: "Nunito", sans-serif;
-$font-size-base: 0.9rem;
-$line-height-base: 1.6;
+// $font-family-sans-serif: "Nunito", sans-serif;
+// $font-size-base: 0.9rem;
```
Comment (404tonynotfound) [inline resources/sass/_variables.scss]: lets just remove these commented lines

File: resources/sass/app.scss [hunk 5]
```diff
@@ -1,14 +1,17 @@
 
 // Fonts
-@import url('https://fonts.googleapis.com/css?family=Nunito');
-
+// @import url('https://fonts.googleapis.com/css?family=Nunito');
+// @import url('https://fonts.googleapis.com/css?family=Roboto:100,300,400,500,700,900|Material+Icons');
```
Comment (404tonynotfound) [inline resources/sass/app.scss]: same here we can remove this

File: resources/views/index.blade.php [hunk 6]
```diff
@@ -3,9 +3,12 @@
 <head>
   <meta charset="utf-8">
   <meta http-equiv="X-UA-Compatible" content="IE=edge,chrome=1">
+  <meta name="csrf-token" content="{{ csrf_token() }}">
     <title>Checkpoint</title>
   <meta name="description" content="The Admin and Supervisor Portal">
+  <link href='https://fonts.googleapis.com/css?family=Roboto:100,300,400,500,700,900|Material+Icons' rel="stylesheet">
```
Comment (404tonynotfound) [inline resources/views/index.blade.php:9]: I don't think we need these fonts right now

**Review by 404tonynotfound — CHANGES_REQUESTED** [review]

**Review by 404tonynotfound — APPROVED** [review]

**Review by JayJayAquino — APPROVED** [review]
👍

**Pull request merged**